components/Academy/OpenToAll/OpenToAll.module.scss

```diff
@@ -0,0 +1,162 @@
+.container {
+    position: relative;
+    margin-bottom: $grid * 2;
+
+    @include media-query(medium) {
+        margin-bottom: 0;
+        padding-bottom: $grid * 2;
+    }
+
+    @include media-query(large) {
+        margin-top: $grid * 16;
+        margin-bottom: $grid * 20;
+        padding-bottom: 0;
+    }
+}
+
+.title {
```

Comment:
**Suggestion**: This will need to support longer titles, maybe `max-width: 660px` will work I'm not sure.
<img width="1107" alt="Screenshot 2022-03-31 at 10 12 52" src="https://user-images.githubusercontent.com/35934884/161021081-ce86cfe9-478c-4fad-937b-1aecdffff02a.png">
;

Reply:
Added this in 👍